doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -153,7 +153,7 @@ driz_sep_bits: int (*default="16"*)
     Integer sum of all the DQ bit values from the input image's DQ array that should be considered 'good' when building the weighting mask. This can also be used to reset pixels to good if they had been flagged as cosmic rays during a previous run of ``AstroDrizzle``, by adding the value 4096 for ACS and WFPC2 data. Please see the section on Selecting the ``Bits`` Parameter for a more detailed discussion.
 
 driz_sep_kernel: str (*default="square"*)
```

Comment:
The description for _driz_sep_kernel_ needs to be changed to make the "default" be "turbo" to be consistent with **adrizzle.help** and **astrodrizzle.help**.